src/main/java/Info.java

```diff
@@ -0,0 +1,54 @@
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.*;
```

Comment:
Try to avoid * imports where possible; it can create an overhead which can impact program execution performance